js/stores/windowStore.js

```diff
@@ -788,8 +790,17 @@ const doAction = (action) => {
         break
       }
     case windowConstants.WINDOW_ON_WINDOW_UPDATE:
-      windowState = windowState.set('windowInfo', action.windowValue)
-      break
+    case appConstants.APP_WINDOW_READY:
```

Comment:
If you did a follow up, this would be fairly easy to add a unit test for too 😄  Basically checking that `focusTime` gets added to `windowInfo`